media-libs/libwebm/libwebm-1.0.0.27.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="The SDK of the VP8 video codec developed by Google"
+DESCRIPTION="libwebm - WebM VP8 Codec SDK"
+HOMEPAGE="https://www.webmproject.org/"
+COPYRIGHT="2010 Google Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/webmproject/libwebm/archive/libwebm-$portVersion.tar.gz"
+CHECKSUM_SHA256="1332f43742aeae215fd8df1be6e363e753b17abb37447190e789299fe3edec77"
+SOURCE_DIR="libwebm-libwebm-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libwebm$secondaryArchSuffix = $portVersion
+	cmd:dumpvtt$secondaryArchSuffix
+	cmd:vttdemux$secondaryArchSuffix
+	lib:libwebm$secondaryArchSuffix
```

Comment:
Should we also add `= $portVersion` to every `cmd:`s and `lib:`s here?

Reply:
IIRC the commands wont produce version output (need to check later (again)) :)

Reply:
Well, the benefit of adding a ` = $portVersion` to the `cmd:` entries is that it allows other recipes to add a version condition to their dependency, in case any of these runtimes has had a bug (in an old release) which prevents that other recipe to run and/or build correctly.
So, even if the runtimes don't report any version, it is still nice to add the version info in the recipe.

Reply:
So should I include them or not? ;)

Reply:
I'd include it.
(I can't see of any downsides.)